src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -386,6 +386,7 @@ public sealed partial class HttpContentHeaders : System.Net.Http.Headers.HttpHea
         public byte[] ContentMD5 { get { throw null; } set { } }
         public System.Net.Http.Headers.ContentRangeHeaderValue ContentRange { get { throw null; } set { } }
         public System.Net.Http.Headers.MediaTypeHeaderValue ContentType { get { throw null; } set { } }
+        internal override void EnsureHeaders() { }
```

Comment:
If it's an internal method, then you don't need to put this declaration into the 'ref' file.  So, delete this line change.